tomb-common/src/banyan_api/models/bucket_key.rs

```diff
@@ -26,12 +26,12 @@ pub struct BucketKey {
 impl Display for BucketKey {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         let status = if self.approved {
-            "approved"
+            "Approved".green()
```

Comment:
woah i didn't even know we could do that

Reply:
it's a really lightweight crate called Colorize and it is helping to make the CLI wayy more legible- and is all still standard output at the end of the day